ADDONS.md

```diff
@@ -116,38 +117,35 @@ module RubyLsp
 
       sig do
         override.params(
+          response_builder: ResponseBuilders::Hover,
           nesting: T::Array[String],
           index: RubyIndexer::Index,
           dispatcher: Prism::Dispatcher,
-        ).returns(T.nilable(Listener[T.nilable(Interface::Hover)]))
+        ).void
       end
-      def create_hover_listener(nesting, index, dispatcher)
+      def create_hover_listener(response_builder, nesting, index, dispatcher)
         # Use the listener factory methods to instantiate listeners with parameters sent by the LSP combined with any
         # pre-computed information in the addon. These factory methods are invoked on every request
-        Hover.new(@config, dispatcher)
+        Hover.new(client, response_builder, @config, dispatcher)
```

Comment:
Let's also mention that it's possible to instantiate multiple listener classes in a comment?